src/Entity/DTO/AlertDTO.php

```diff
@@ -15,80 +15,71 @@
 class AlertDTO
 {
     /**
-     * @var int
      * @IS\Id
      * @IS\Expose
      * @IS\Type("integer")
      */
-    public $id;
+    public int $id;
 
     /**
-     * @var int
      * @IS\Expose
      * @IS\Type("integer")
      */
-    public $tableRowId;
+    public int $tableRowId;
 
     /**
-     * @var int
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $tableName;
+    public int $tableName;
 
     /**
-     * @var int
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $additionalText;
+    public int $additionalText;
```

Comment:
this should be a string `?string $additionalText`, please compare to corresponding entity.